modules/targetAcquisition/targetAcquisition.py

```diff
@@ -45,8 +46,17 @@ def __init__ (self, deckLinkFrame=np.zeros((416, 416, 3))):
         """
         self.boxes = [] # Contains BoundBox objects (see utils.py), each of which contains opposite corners of a rectangle by percentage of height and width of the image as (xmin, ymin) to (xmax, ymax)
         self.tentCoordinates = dict()
-        self.currentFrame = deckLinkFrame
-    
+        self.currentFrame = np.array()
+        self.videoPipeline = videoPipeline
+        self.coordinatePipeline = coordinatePipeline
+        main = threading.Thread(target=self._main_())
```

Comment:
Improve naming (see other comment). I was going to say to add a comment but if the name makes what's happening clear then it's a good name. Right now it is not a good name.

Reply:
Yeah I made this mistake a lot, good catch